src/Microsoft.ML.Core/CommandLine/CmdParser.cs

```diff
@@ -462,7 +462,7 @@ private static ArgumentInfo GetArgumentInfo(Type type, object defaults)
         {
             Contracts.CheckValue(type, nameof(type));
             Contracts.CheckParam(type.IsClass, nameof(type));
-
+            var fullname = type.FullName;
             var args = new List<Argument>();
```

Comment:
This doesn't seem to be used anywhere? Is it necessary?

Reply:
Nope, good catch. It was used for debugging. I'll remove it.